src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  }
+
+  return (days * 40) - 50;
```

Comment:
Let's move mysterious numbers to variables with self-describing names.
What is 40? What is 20? What is 50?